test/fixtures/default/.psysh.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/*
+ * This file is part of Psy Shell
+ *
+ * (c) 2015 Justin Hileman
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+return [];
```

Comment:
:cry: 5.3 doesn't like the shorthand syntax. https://travis-ci.org/bobthecow/psysh/jobs/76518034